src/plugin/transform-node.ts

```diff
@@ -0,0 +1,33 @@
+import {transform as baseTransform} from './transform';
+import {PluginOptions} from './types';
+
+function copy(from: string, to: string) {
+    const {mkdirSync, copyFileSync} = dynrequire('node:fs');
+    const {dirname} = dynrequire('node:path');
+
+    mkdirSync(dirname(to), {recursive: true});
+    copyFileSync(from, to);
+}
+
+/*
+ * Runtime require hidden for builders.
+ * Used for nodejs api
+ */
+export function dynrequire(module: string) {
```

Comment:
This is not necessary now. If we create a separate layer for node, we can use a common require here.